Makefile

```
@@ -350,3 +350,13 @@ release-changelog: $(wildcard changelog/*.txt)
 	@:$(if $(LAST_RELEASE),,$(error please set the LAST_RELEASE environment variable for changelog generation))
 	@:$(if $(THIS_RELEASE),,$(error please set the THIS_RELEASE environment variable for changelog generation))
 	changelog-build -changelog-template changelog/changelog.tmpl -entries-dir changelog -git-dir . -note-template changelog/note.tmpl -last-release $(LAST_RELEASE) -this-release $(THIS_RELEASE)
+
+.PHONY: dev-gorelease
+dev-gorelease: export GORELEASER_PREVIOUS_TAG := $(shell git describe --tags $(shell git rev-list --tags --max-count=1))
+dev-gorelease: export GORELEASER_CURRENT_TAG := v999.999.999
```

Instead of setting this to a nonsense number, I'd prefer if we do something like this
```suggestion
dev-gorelease: export GORELEASER_CURRENT_TAG := $(shell git describe --tags --exclude "api/*" --exclude "sdk/*" )
```
It will result in a tag like `v2.0.0-115-g1c0c5647bc`